tempesta_fw/http.c

```diff
@@ -4962,6 +4962,11 @@ tfw_http_get_ip_from_xff(TfwHttpReq *req)
 	 * IP address by the first value in X-Forwarded-For
 	 */
 	s_xff = req->h_tbl->tbl[TFW_HTTP_HDR_X_FORWARDED_FOR];
+	/*
+	 * Get only the first duplicate header
+	 */
```

Comment:
```suggestion
	 /* Pick only end client address, ignore all proxies between it and us.  */
```
First, don't explain what expressions do, explain why they do that.
Second, it's just a single line comment, don't need to extend it to take three lines.